### ISSUE TITLE
"'steamwebhelper' is not responding" error on Fedora 39 KDE Plasma spin

### ISSUE THREAD

**Issue by GhostCoder22** [ValveSoftware/steam-for-linux#10606]
[The following is system information provided by "Info Center"]
Distribution: Fedora 39
KDE Plasma Version: 5.27.10
KDE Frameworks Version: 5.115.0
Qt Version: 5.15.12
Kernel Version: 6.7.7-200.fc39.x86_64 (64-bit)
Graphics Platform: Wayland
Processors: 4 × Intel® Core™ i3-7100U CPU @ 2.40GHz
Memory: 15.4 GiB of RAM
Graphics Processor: Mesa Intel® HD Graphics 620
Storage Device: 1 TB 5200 RPM Hard Drive.

Steam Version: 1709846872
Steam Beta Branch: Stable Client
A system update occurred yesterday.
I have not opted into Steam client beta.
My GPU is Intel.

Since my chosen Linux distro is Fedora and after reporting the following directly through Steam for Linux last night, I was advised to report bugs here citing Steam for Linux "is only supported on the latest version of 64-bit Ubuntu LTS with Unity, Gnome, or KDE desktops".

Anyways, I'd like to report a crash when starting Steam for Linux lasdt night, this one pertaining Steam's steamwebhelper component. My chosen operating system is Fedora 39 KDE Plasma Spin.

Just after a system update and rebooting my laptop, I went to start Steam only to find the error

> "steamwebhelper, a critical Steam component, is not responding. The Steam UI will not be usable."

Out of the 5 options, I've chosen option #1 "Restart Steam", which did cause Steam to restart normally.

This error has occurred once or twice previously after completely replacing Windows 10 with Fedora.

I don't have any security software installed, nor do I use VPN except on very rare occasions (but I currently don't have any VPN software installed either).

#### I do not know the steps for reproducing this issue at this time

[steam-logs.tar.gz](https://github.com/ValveSoftware/steam-for-linux/files/14551962/steam-logs.tar.gz)
![steamwebhelper unresponsive](https://github.com/ValveSoftware/steam-for-linux/assets/13742124/eac9ffc1-0d71-46e1-a3e2-cca6e05d5e5f)

**Comment by GhostCoder22** [comment]
I've just had yet another crash, and this time I managed to find the **steamwebhelper.log** file on my Fedora distro installation to be examined.

[steamwebhelper.log](https://github.com/ValveSoftware/steam-for-linux/files/14964345/steamwebhelper.log)

**Comment by GhostCoder22** [comment]
steamwebhelper crashed yet again, but this time after updating Steam. Here is yet another steamwebhelper crash log:
[steamwebhelper.log](https://github.com/ValveSoftware/steam-for-linux/files/15449624/steamwebhelper.log)

**Comment by smcv** [comment]
> Would whatever has been causing these unexpected crashes like the one reported here also cause these random intermittent "'steamwebhelper' is not responding" errors?

Yes. If the `steamwebhelper` crashes, for any reason, then that will have two effects:

1. Steam will no longer be able to communicate with the `steamwebhelper`. The Steam user interface will disappear or freeze, and eventually the "'steamwebhelper' is not responding" error message will pop up.
2. If your OS has a crash reporting mechanism, it will see the crash and respond to it by trying to report it.

In your more recent logs, the situation looks similar to what I said in https://github.com/ValveSoftware/steam-for-linux/issues/10606#issuecomment-1991781383, although with different messages logged. The container runtime is starting up correctly, and `steamwebhelper` starts, but then it crashes.

(That means I am unable to help with this further, because the part of the process that I'm involved with developing is working correctly; probably only a `steamwebhelper` developer within Valve would be able to diagnose or solve `steamwebhelper` crashes.)

**Comment by DanMan** [comment]
Steam ("steamwebhelper killed by SIGTRAP") keeps crashing for me on Fedora 41 w/ Nvidia, too. It drags the whole Gnome Shell down with it. When it happens, it's always when I re-focus the Steam client with the mouse while it's sitting behind my currently active window. Has been happening for weeks if not months now.

Steam Beta Branch:  Steam Beta Update
Steam Version:  1734743608
Steam Client Build Date:  Sat, Dec 21 02:01 UTC -08:00
Steam Web Build Date:  Fri, Dec 20 22:57 UTC -08:00
Steam API Version:  SteamClient021

**Comment by GhostCoder22** [comment]
Since the last bug report I tried sending in [https://github.com/ValveSoftware/steam-for-linux/issues/11699](url) was closed for duplicating this bug report, I'd like to try reporting this continuously ongoing bug pertaining only to **steamwebhelper** yet again especially since I have yet to receive any kind of help for it.

- Steam client version (build number or date): **1743554648**
- Steam Client Build Date: **Tue, Apr 1 17:27 UTC -08:00**
- Steam Web Build Date: **Tue, Apr 1 17:34 UTC -08:00**
- Distribution (e.g. Ubuntu): Fedora
- Opted into Steam client beta?: No

Have you checked for system updates?: Yes

Steam Logs: [steam-logs-20250416.tar.gz](https://github.com/user-attachments/files/19783101/steam-logs-20250416.tar.gz)

- KDE Plasma Version: 6.3.4
- KDE Frameworks Version: 6.13.0
- Qt Version: 6.8.2
- Kernel Version: 6.13.10-200.fc41.x86_64 (64-bit)
- Graphics Platform: Wayland
- Processors: 4 × Intel® Core™ i3-7100U CPU @ 2.40GHz
- Memory: 15.4 GiB of RAM
- Graphics Processor: Mesa Intel® HD Graphics 620

Last year I reported an https://github.com/ValveSoftware/steam-for-linux/issues/10606#issuecomment-2260305391 but unfortunately nothings really been done to resolve all these steamwebhelper crashes dispite the number of steamwebhelper crash reports is issued, which to me is a bit concerning.

This time, the usual Steamwebhelper is not responding dialog appeared after Steam for Linux updated itself. Valve, I beg you to fix all the steamwebhelper crashes.

Please Valve, I beg of you to please fix all of these dang steamwebhelper crashes that's been plaguing Steam for Linux users.
